src/garage/replay_buffer/meta_replay_buffer.py

```diff
@@ -0,0 +1,145 @@
+"""A replay buffer memory for meta RL."""
```

Comment:
@krzentner please take a look here to make sure this is consistent with your API for samples vs paths

Reply:
So, the design of this replay buffer is fine, but I don't understand why it's necessary at all.

What does this buffer implement that `PathBuffer` doesn't?
If it's something small (like adding individual samples), then I can just add it to `PathBuffer`.
If it's something really big I somehow can't see in this code, it really needs to be called out in the class docstring.

Reply:
It does mostly the same thing as PathBuffer but with a different structure of how data is stored. It has the `random_sequence` method which samples trajectories, and it's called when a recurrent encoder is used in PEARL. I took this from the oyster repo.

Reply:
@krzentner please make the call here